plc4go/protocols/modbus/readwrite/ParserHelper.go

```diff
@@ -40,7 +40,7 @@ func (m ModbusParserHelper) Parse(typeName string, arguments []string, io utils.
 		if err != nil {
 			return nil, errors.Wrap(err, "Error parsing")
 		}
-		return model.DataItemParse(io, dataType, numberOfValues)
+		return model.DataItemParse(io, dataType, numberOfValues, 0)
```

Comment:
This is a generated file and will be replaced by an updated version, effectively reverting your changes the next time "mvn install" is executed in the parent.